src/components/EditImageModal/index.jsx

```diff
@@ -404,10 +530,46 @@ export default class EditImageModal extends React.Component {
       imageSource: event.detail.src || '',
       imageStyle: event.detail.style || '',
       isImageDecorative: event.detail.alt === '',
+      // if existing img had a source, assume it could be loaded and show the image preview
```

Comment:
I'm not sure whether this will cause some funkiness with an edge case I couldn't think of, but it's the best way I could ensure that an image preview is shown again when editing an existing image.